systemProperties.adoc

```diff
@@ -0,0 +1,24 @@
+== System Properties
+Below, a number of relevant System Properties are listed.
+
+
+NOTE: All boolean properties below are read using link:https://javadoc.io/static/net.openhft/chronicle-core/2.23ea13/net/openhft/chronicle/core/Jvm.html#getBoolean-java.lang.String-[net.openhft.chronicle.core.Jvm.getBoolean(java.lang.String)], and so are enabled if either `-Dflag` or `-Dflag=true` or `-Dflag=yes`
+
+.System properties
+[cols=4*, options="header"]
+|===
+| Property Key | Default | Description | Java Variable Name (Type)
+| chronicle.network.callOnMissedHeartbeatOnDisconnect |`false` | If enabled, calls onMissedHeartbeat when the connection is abruptly dropped | _CALL_MISSED_HEARTBEAT_ON_DISCONNECT_ (boolean)
+| client.timeout | 500 | The overall timeout; number of milliseconds after which a client is timed out if they cannot connect to a socket | _pausePeriodMs_ (int)
+| connectionStrategy.pauseMillisBeforeReconnect | 500 | Allows control of a backoff strategy, and returns how long in milliseconds to pause before attempting reconnect | pauseMillisBeforeReconnect (long)
+| connectionStrategy.pause.max.secs | 5 | Controls the maximum time the ConnectionStrategy.connect method is paused for | _maxPauseSec_ (long)
+| connectionStrategy.pause.min.secs | 5 | Controls the minimum time the ConnectionStrategy.connect method is paused for | _minPauseSec_ (long)
+| connectionStrategy.socketConnectionTimeoutMs | 1 | Amount of time waiting for a socket to open i.e. if the strategy is to retry every x seconds, each individual attempt will timeout after `connectionStrategy.socketConnectionTimeoutMS` and it will stop retrying after `client.timeout` | socketConnectionTimeoutMs (int)
+| disable.tcp.nodelay | `false` | This disables link:https://en.wikipedia.org/wiki/Nagle%27s_algorithm[this] for connections. When true, the connection will optimise for low packet latency at the cost of overall traffic | _DISABLE_TCP_NODELAY_ (boolean)
```

Comment:
I think the link text should be "Nagle's algorithm" because anyone disabling it should probably know what it is, and "This disables this" reads a bit weird.